app/models/requests/requestable_decorator.rb

```diff
@@ -139,7 +139,7 @@ def help_me_message
             else
               "digital_access"
             end
-      I18n.t("requests.help_me.brief_msg.#{key}_html")
+      I18n.t("requests.help_me.brief_msg.#{key}_html").html_safe # rubocop:disable Rails/OutputSafety
```

Comment:
It seems like the `.html_safe` won't be needed once we get to Rails 7 (see https://github.com/rails/rails/pull/43415).  But this seems like the best approach until then.